lib/cache/cache.go

```diff
@@ -90,9 +90,9 @@ func ForAuth(cfg Config) Config {
 		{Kind: types.KindReverseTunnel},
 		{Kind: types.KindTunnelConnection},
 		{Kind: types.KindAccessRequest},
-		{Kind: types.KindAppServer},
+		{Kind: types.KindAppServer, SubKind: types.SubKindTeleportApp},
+		{Kind: types.KindAppServer, SubKind: types.SubKindOktaApp},
```

Comment:
What's this for?

Reply:
I think I was overthinking this. Was effecitvely trying to limit different app subkinds to specific processes, but I think that's overkill.